distributed/scheduler.py

```diff
@@ -6111,13 +6111,13 @@ def reevaluate_occupancy(self, worker_index=0):
             if self.status == Status.closed:
                 return
 
-            last = time()
-            next_time = timedelta(seconds=DELAY)
-
             if self.proc.cpu_percent() < 50:
-                workers = list(self.workers.values())
-                for i in range(len(workers)):
-                    ws: WorkerState = workers[worker_index % len(workers)]
+                last = time()
```

Comment:
FWIW this takes `~15us`. So is insignificant at the level milliseconds, which is what we compare `duration` with later.

```python
In [1]: import psutil

In [2]: proc = psutil.Process()

In [3]: %%timeit
   ...: if proc.cpu_percent() < 50:
   ...:     pass
   ...: 
14.9 µs ± 139 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```